src/native/libs/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -354,8 +354,10 @@ extern bool g_libSslUses32BitTime;
     REQUIRED_FUNCTION(EC_GROUP_get_degree) \
     REQUIRED_FUNCTION(EC_GROUP_get_order) \
     REQUIRED_FUNCTION(EC_GROUP_get_seed_len) \
+    LIGHTUP_FUNCTION(EC_GROUP_get_field_type) \
```

Comment:
You'll need to define the prototype in osslcompat_30.h so it can be built against the 1.1.x headers.